### PR TITLE
Update the documentation setup

### PR DESCRIPTION
**Pull request by ChrisRackauckas**
This sets everything up as built doc examples, splits tutorials from examples, adds more informative titles, and sets up buildkite

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
@thazhemadam can you help get the docs build over to buildkite?

@ashutosh-b-b Can you work through any of the failures that come up with the docs builds being fully enabled?

File: docs/pages.jl [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-    "Solver Algorithms" => ["MLP.md",
-        "DeepSplitting.md",
-        "DeepBSDE.md"],
     "Tutorials" => [
         "tutorials/deepsplitting.md",
         "tutorials/deepbsde.md",
         "tutorials/mlp.md",
     ],
+    "Extended Examples" => [
+        "examples/blackscholes",
```
Comment (ChrisRackauckas) [inline docs/pages.jl:10]:
```suggestion
        "examples/blackscholes.md",
```

File: docs/src/tutorials/deepbsde.md [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
-<!-- TODO: revise the code, which should not work right here -->
-### Solving a 100-dimensional Hamilton-Jacobi-Bellman Equation
+# Solving a 100-dimensional Hamilton-Jacobi-Bellman Equation with `DeepBSDE`
 
 First, here's a fully working code for the solution of a 100-dimensional
 Hamilton-Jacobi-Bellman equation that takes a few minutes on a laptop:
 
-```julia
+```@example deepbsde
 using NeuralPDE
```
Comment (ChrisRackauckas) [inline docs/src/tutorials/deepbsde.md:7]:
```suggestion
using HighDimPDE
```

**Comment by thazhemadam** [comment]
This has been superseded by #96 in which @ashutosh-b-b has also completed it, with the changes introduced here included.

**Pull request closed without merging**